plugins/heindekort/extract.js

```diff
@@ -0,0 +1,5 @@
+function(page) {
+    var regex = /<img[^>]*src="([^"]*)"/;
```

Comment:
This is the other extreme of the previous regex you had.
The previous one was too specific, this one is way too generic since it matches any image.
If the website decides to change slightly the layout and adds an image in the header, the strip extraction would break.
Please match at least one relevant part of the image path.